algorithms/Fibonacci series.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+int main()
+{
+  int n, first = 0, second = 1, next, c;
```

Comment:
Your code prints garbage values when the user specifies the ``n`` value greater than 47.
That's because ``int`` can no longer hold values higher than 2147483648 where the Fibonacci numbers shoot off exponentially.
Please fix this.